docs/customize/static_pages.md

```diff
@@ -32,7 +32,7 @@ The templates contain the content that will be displayed in the static pages.
 Static pages are loaded during the setup (`invenio-cli services setup`). But can also be loaded any time by using the following command.
 
 ```bash
-pipenv run invenio rdm-records pages create
+pipenv run invenio rdm pages create
```

Comment:
I understand this is a separate command, in addition to `rdm fixtures` correct ?

Reply:
Yes, `rdm fixtures`, creates custom OAI sets and pages (meant to be used only during setup), while this one allows to re-create the pages if needed any time.